### PR TITLE
Changed latex strings to raw strings

### PR DESCRIPTION
**Pull request by Ak0303**
## Description
I've changed all the latex strings to raw strings !!

## Motivation and Context

  #1206 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have assigned/requested two reviewers for this pull request.

### CODE REVIEW

**Comment by jaladh-singhal** [comment]
Thanks for this PR @Ak0303 - we'll soon review it and let you know if it requires any edits! Keep contributing :tada:

File: tardis/plasma/properties/ion_population.py [hunk 1]
```diff
@@ -42,12 +42,11 @@ class PhiSahaLTE(ProcessingPlasmaProperty):
...
     outputs = ("phi",)
-    latex_name = ("\\Phi",)
+    latex_name = (r"\Phi",)
     latex_formula = (
-        "\\dfrac{2Z_{i,j+1}}{Z_{i,j}}\\Big(\
-                     \\dfrac{2\\pi m_{e}/\\beta_{\\textrm{rad}}}{h^2}\
-                     \\Big)^{3/2}e^{\\dfrac{-\\chi_{i,j}}{kT_{\
-                     \\textrm{rad}}}}",
+        r"\dfrac{2Z_{i,j+1}}{Z_{i,j}}\Big(\
+                     dfrac{2\pi m_{e}/\beta_{\textrm{rad}}}{h^2}\
```
Comment (Rodot-) [inline tardis/plasma/properties/ion_population.py:48]: Shouldn't there be a '\' before dfrac?

File: tardis/plasma/properties/ion_population.py [hunk 2]
```diff
@@ -42,12 +42,11 @@ class PhiSahaLTE(ProcessingPlasmaProperty):
...
+                     Big)^{3/2}e^{\dfrac{-\chi_{i,j}}{kT_{\textrm{rad}}}}",
```
Comment (Rodot-) [inline tardis/plasma/properties/ion_population.py:49]: And here as well, before Big

File: tardis/plasma/properties/ion_population.py [hunk 3]
```diff
@@ -99,12 +98,11 @@ class ThermalPhiSahaLTE(PhiSahaLTE):
...
     outputs = ("thermal_phi_lte",)
-    latex_name = ("\\Phi^{*}(T_\\mathrm{e})",)
+    latex_name = (r"\Phi^{*}(T_\mathrm{e})",)
     latex_formula = (
-        "\\dfrac{2Z_{i,j+1}}{Z_{i,j}}\\Big(\
-                     \\dfrac{2\\pi m_{e}/\\beta_{\\textrm{electron}}}{h^2}\
-                     \\Big)^{3/2}e^{\\dfrac{-\\chi_{i,j}}{kT_{\
-                     \\textrm{electron}}}}",
+        r"\dfrac{2Z_{i,j+1}}{Z_{i,j}}\Big(\
+                     dfrac{2\pi m_{e}/\beta_{\textrm{electron}}}{h^2}\
```
Comment (Rodot-) [inline tardis/plasma/properties/ion_population.py:104]: Again, here

**Review by Rodot- — COMMENTED** [review]
Have you verified yourself that these changes work?  Can you put in some screenshots of all of the rendered LaTeX?

**Review by jaladh-singhal — COMMENTED** [review]
Apparently every change looks fine to me!

@Ak0303 can you still cross-check if your changes are rendered correctly (like @Rodot- suggested) by rendering each class property you changed (`latex_name` or `latex_formula`) as Latex in Jupyter notebook. E.g. for `Levels` class in `tardis/plasma/properties/atomic.py`, it'll look like:
```
import tardis
from IPython.display import Math, display
for ln in tardis.plasma.properties.atomic.Levels.latex_name:
    display(Math(ln))
```
The output should be exact same for your modified code and current code at tardis.

**Pull request merged**